perfact/zodbsync/zodbsync.py

```diff
@@ -582,21 +586,26 @@ def playback(self, path=None, recurse=True, override=False,
             parent_obj.manage_delObjects(ids=[part])
             return
 
-        fs_data = self.fs_read(path, encoding=encoding)
+        fs_data, fs_raw_meta = self.fs_read(path, encoding=encoding)
         if 'unsupported' in fs_data:
             self.logger.warning('Skipping unsupported object ' + path)
             return
 
         try:
             srv_data = (
-                dict(mod_read(obj, default_owner=self.manager_user))
-                if obj_exists else None
+                mod_read(obj, default_owner=self.manager_user) if obj_exists
+                else None
             )
         except Exception:
             self.logger.exception('Unable to read object at %s' % path)
             raise
 
-        if fs_data != srv_data:
+        # compare source code as raw if existent
+        srv_meta = mod_format(srv_data, ignore_source=True)
```

Comment:
I think this is the source of the problem. `srv_data` is `None` if the object does not yet exist. `mod_format` does not handle this. The earlier comparison `fs_data != srv_data` did.